samples/highcharts/coloraxis/width-and-height/demo.js

```diff
@@ -0,0 +1,11 @@
+Highcharts.chart('container', {
+    colorAxis: {
+        width: '50%',
+        height: 80
+    },
+
```

Comment:
Add a `title` to, so random viewers get some more information. "Color axis with specified width and height" would be a nice fit here.